e2e-tests/playwright/lib/src/server/default_config.ts

```diff
@@ -815,5 +812,12 @@ const defaultServerConfig: AdminConfig = {
             ReviewerCommentRequired: true,
             HideFlaggedContent: true,
         },
+        ReviewerSettings: {
+            CommonReviewers: true,
+            CommonReviewerIds: [],
+            TeamReviewersSetting: {},
+            SystemAdminsAsReviewers: false,
+            TeamAdminsAsReviewers: true,
+        },
```

Comment:
Note: Though same, this change was intended to match the output `config.json` (default config) generated by `make config-reset`